### ISSUE TITLE
Stream only updating once per second

### ISSUE THREAD

**Issue by vanlooverenkoen** [hemanthrajv/flutter_compass#77]
I am only receiving an update once a second. This is not usable to create a real-time compas.

**Comment by vanlooverenkoen** [comment]
https://github.com/hemanthrajv/flutter_compass/pull/68
This fixed it

**Comment by vanlooverenkoen** [comment]
```
dependency_overrides:
  flutter_compass:
    git:
      url: git@github.com:hemanthrajv/flutter_compass.git
      ref: 48e155bc647593e588fdeb5b22ed066afc1275ef
```

**Comment by vanlooverenkoen** [comment]
@hemanthrajv is it possible to create a new release on pub.dev

**Comment by gosusnjar** [comment]
I tried using this and got git access error. I solved it with using https. (Answer found here https://githubmemory.com/repo/trevorwang/retrofit.dart/issues/345, last comment))
```
dependency_overrides:
  flutter_compass:
    git:
      url: https://github.com/hemanthrajv/flutter_compass.git
      ref: 48e155bc647593e588fdeb5b22ed066afc1275ef
```

**Comment by vanlooverenkoen** [comment]
> I tried using this and got git access error. I solved it with using https. (Answer found here https://githubmemory.com/repo/trevorwang/retrofit.dart/issues/345, last comment))
> 
> ```
> dependency_overrides:
>   flutter_compass:
>     git:
>       url: https://github.com/hemanthrajv/flutter_compass.git
>       ref: 48e155bc647593e588fdeb5b22ed066afc1275ef
> ```

That just means that your ssh keys are not setup correctly

**Comment by gosusnjar** [comment]
> That just means that your ssh keys are not setup correctly

Oh, good to know, thanks.

**Comment by vanlooverenkoen** [comment]
Fixed in #68

**Comment by vanlooverenkoen** [comment]
Create a new ticket #80 so we can get this updated to pub.dev
